integration_tests/requirements.txt

```diff
@@ -1,3 +1,3 @@
-dbt==0.20.0
-dbt-spark==0.20.0
-dbt-spark[PyHive]==0.20.0
+dbt==0.20.1rc1
```

Comment:
oo why the release candidate? bc of spark stuff?

Reply:
Yup! dbt-spark 0.20.0 has an error with the get_column_values() macro and seems to be unstable in these instanced. The releases candidate fixes this. dbt-spark 0.20.1 is released, but it is dependent on dbt 0.20.1 which is not released yet. So we shall use the release candidate in the meantime.